2200054/소득 불균형.py

```diff
@@ -0,0 +1,16 @@
+import sys
+input = sys.stdin.readline
+
+TC = int(input())
+for t in range(1, TC+1):
+    N = int(input()); inputnumbers = list(map(int, input().split())); numbers = []
+
+    for i in range(N): # 숫자를 N개 초과해서 입력햇을때를 대비한 리스트 재 설정.
+        numbers.append(inputnumbers[i])
+    avr = sum(numbers) // len(numbers)
```

Comment:
```suggestion
    avr = sum(numbers) / len(numbers)
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄